spec/tasks/whitehall_importer_spec.rb

```diff
@@ -45,6 +55,17 @@
     expect(edition.update_type).to eq("major")
   end
 
+  it "adds users who have never logged into Content Publisher" do
+    importer = Tasks::WhitehallImporter.new(123, import_data)
+    importer.import
+
+    expect(User.last.uid).to eq "36d5154e-d3b7-4e3e-aad8-32a50fc9430e"
+    expect(User.last.name).to eq "A Person"
+    expect(User.last.email).to eq "a-publisher@department.gov.uk"
+    expect(User.last.organisation_slug).to eq "a-government-department"
+    expect(User.last.organisation_content_id).to eq "01892f23-b069-43f5-8404-d082f8dffcb9"
+  end
```

Comment:
Is it worth adding a test that users who _have_ logged into Content Publisher already don't get re-added?